backend/frontend/templates/frontend/progress_details/page.html

```diff
@@ -127,7 +127,7 @@ <h6>Link to your work</h6>
                             <ul class="list-disc">
                                 {% for pr in course_component.repository.pull_requests.all %}
                                     <li class="ml-4 mb-1">
-                                        <a id="pr_{{pr.number}}" class="{{ styles.link }} underline" href="{{ course_component.repository.get_github_repo_link }}/pull/{{ pr.number }}" target="_blank">{{ pr.title }} - {{ pr.created_at }}</a>
+                                        <a id="pr_{{pr.number}}" class="{{ styles.link }} underline" href="{{pr.get_github_pr_link}}" target="_blank">{{ pr.title }} - {{ pr.created_at }}</a>
```

Comment:
Sorry to be a stickler for such a small thing, but if we've put it on the PullRequest model, then adding "pr" in the name seems redundant. Can it just be "get_github_link()" instead?